cfpb/cfgov-refresh
# Housing counselor: Add links for phone and email addresses

Phone numbers should probably be clickable for mobile users.
Email addresses should probably be as well.

## Changes

- Makes search result item phone numbers `tel:` URL links.
- Makes search result email address `mailto:` URL links.

## Testing

1. Visit https://www.consumerfinance.gov/find-a-housing-counselor/?zipcode=12047 and see that phone and email are not links in the results.
2. Pull this branch and visit http://localhost:8000/find-a-housing-counselor/?zipcode=12047 and see that phone and emails are links in the results list.

## Screenshots

<img width="296" alt="screen shot 2018-12-11 at 10 35 44 am" src="https://user-images.githubusercontent.com/704760/49811101-8c5d9180-fd30-11e8-924b-2b128a3bca77.png">


## Notes

- I added `tel:+1-` to add a "+1-" to the beginning of the number. I'm not sure if this is necessary, but I saw it elsewhere. However, this will be an error if any of the data has a country code in the number. Any way we can check that? (cc @schbetsy)
